### ISSUE TITLE
Update Large Projects wiki page

### ISSUE THREAD

**Issue by FezVrasta** [adobe/brackets#11193]
You should replace 

https://github.com/gruehle/exclude-folders

with

https://github.com/JonathanWolfe/file-tree-exclude

inside this page

https://github.com/adobe/brackets/wiki/Large-Projects

**Comment by redmunds** [comment]
Good idea, but I think *both* should be mentioned and let user decide.

**Comment by FezVrasta** [comment]
I thought the old one doesn't work anymore, this is what is written in the file-tree-exclude readme:

> This is an updated version of Glenn Ruehle's now defunkt plugin - exclude-folders.

If the old one still works then I will ask @JonathanWolfe to fix his readme.

**Comment by JonathanWolfe** [comment]
@FezVrasta - Glenn's still works, but is not as accurate or user friendly as mine (or the other forks people have made), and should be depreciated as an option.

Once Brackets get's it's native `exclude-folders` working, both our plugins will be obsolete.

**Comment by redmunds** [comment]
@FezVrasta You're right, @gruehle 's extension no longer seems to work. Someone should open an issue on that repo.

I thought the only difference was that @JonathanWolfe 's extension was preferences driven (which is very cool!)

**Comment by redmunds** [comment]
Yes, @gruehle 's extension still works -- it must have been conflicting with one of my other extensions.
